docs/rescript-usage.md

```
@@ -63,11 +63,13 @@
 - [Transforms](#transforms)
   - [`to` with `~custom`](#to-with-custom)
 - [Functions on schema](#functions-on-schema)
+  - [At a glance](#at-a-glance)
   - [Pipelines](#pipelines)
   - [Built-in operations](#built-in-operations)
   - [`parser` / `asyncParser`](#parser-asyncparser)
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)